lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.h

```diff
@@ -32,7 +33,13 @@ class MAYAUSD_CORE_PUBLIC UsdUndoCreateGroupCommand : public Ufe::UndoableComman
 public:
 	typedef std::shared_ptr<UsdUndoCreateGroupCommand> Ptr;
 
+#if UFE_PREVIEW_VERSION_NUM < 2017
 	UsdUndoCreateGroupCommand(const UsdSceneItem::Ptr& parentItem, const Ufe::PathComponent& name);
+#else
+	typedef std::list<std::shared_ptr<Ufe::UndoableCommand>> UfeUndoCommands;
```

Comment:
As per other comments, if you derived from CompositeUndoableCommand, you wouldn't need this.